.rubocop_todo.yml

```diff
@@ -28,7 +28,7 @@ Metrics/BlockNesting:
 # Offense count: 1
 # Configuration parameters: CountComments.
 Metrics/ClassLength:
-  Max: 161
+  Max: 162
```

Comment:
necessary?

Reply:
Yeah, otherwise rubocop complains, alternatively could try and reduce number of lines in abstract_job.rb but there's not much room for that ( and it doesn't include whitespace )